etcdctlv3/command/range_command.go

```diff
@@ -48,13 +58,40 @@ func rangeCommandFunc(cmd *cobra.Command, args []string) {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
+
+	sortByOrder := pb.RangeRequest_NONE
+	switch {
+	case sortOrder == "ASCEND":
+		sortByOrder = pb.RangeRequest_ASCEND
+	case sortOrder == "DESCEND":
+		sortByOrder = pb.RangeRequest_DESCEND
```

Comment:
curl gives:
1 for `curl foo://foo` (bad protocol)
2 for `curl --foo`, `curl` (bad args, bad flags)
4 for `curl --engine foo http://example.com` (feature not supported)
6 for `curl foo` (can't resolve)
So something like `ErrBadFeature` seems closest to what curl is doing